queries/pkl/highlights.scm

```diff
@@ -71,24 +67,24 @@
 
 (interpolationExpr
   "\\(" @punctuation.special
-  ")" @punctuation.special) @none
+  ")" @punctuation.special) @embedded
 
 (interpolationExpr
  "\\#(" @punctuation.special
- ")" @punctuation.special) @none
-  
+ ")" @punctuation.special) @embedded
+
 (interpolationExpr
   "\\##(" @punctuation.special
-  ")" @punctuation.special) @none
+  ")" @punctuation.special) @embedded
```

Comment:
What is this capture, Neovim doesn't have this capture

Reply:
Thank you so much for your review, I will fix it.
Most of 'highlights.scm' commit contents were grabbed from upstream tree-sitter-pkl, and I thought this one was used for something.